assets/js/frontend/cart.js

```diff
@@ -67,7 +67,7 @@ jQuery( function( $ ) {
 		var $html       = $.parseHTML( html_str );
 		var $new_form   = $( '.woocommerce-cart-form', $html );
 		var $new_totals = $( '.cart_totals', $html );
-		var $notices    = $( '.woocommerce-error, .woocommerce-message, .woocommerce-info', $html );
+		var $notices    = $( '.entry-content', $html ).find( '.woocommerce-error, .woocommerce-message, .woocommerce-info' );
```

Comment:
Not all themes use `.entry-content` class, so it will break compatibility with any theme that already supports WooCommerce, need to find another HTML class.

Reply:
Hi, i think that `		var $notices    = $( 'div.woocommerce-notices-wrapper:last', $html ).find( '.woocommerce-error, .woocommerce-message, .woocommerce-info' );
`
should work on all themes.

Reply:
@claudiosanches fixed as described above.